charts/sorry-cypress/templates/deployment-director.yml

```diff
@@ -17,7 +17,11 @@ spec:
       - env:
         {{- range .Values.director.ingress.hosts }}
         - name: director_URL
-          value: {{ .host }}
+          value: http://{{ .host }}
+        {{- end }}
+        {{- range .Values.dashboard.ingress.hosts }}
+        - name: DASHBOARD_URL
+          value: http://{{ .host }}
```

Comment:
@tico24, in this case, we are expecting the internal URL, right?

Reply:
External I believe. That's how its set up in the normal k8s yaml

Reply:
So should I add a `director.environmentVariables.dashboardUrl` too?